docs/docsite/rst/network/getting_started/first_inventory.rst

```diff
@@ -0,0 +1,253 @@
+***********************************************
+Build Your Inventory
+***********************************************
+
+A fully-featured inventory file can serve as the source of truth for your network. Using an inventory file, a single playbook can maintain hundreds of network devices with a single command. This page shows you how to build an inventory file, step by step.
```

Comment:
We already have quite sophisticated documentation about building inventories in the [general sections](http://docs.ansible.com/ansible/latest/intro_inventory.html).
I'd suggest to link here to the existing documentation for further details and concentrate on networking specific details here.

Reply:
In general I agree that we should duplicate documentation as infrequently as possible. And the existing inventory docs are good. However, the Network Getting Started Guide is designed to walk network admins through the basics, step by step and with examples that look familiar to them (i.e. network devices instead of servers). My sense is that new/potential Ansible Network users need more handholding than the typical Ansible user (developer, sysadmin), so some duplication here is helpful.